### PR TITLE
Resolves #1883 - Purging Squids

### LINKED ISSUE #1883: Squids are not affected by /mp
Thread: http://totalfreedom.boards.net/thread/46093/squids-affected-mobpurge

Steven's Reply:
You are correct. The code for /mp includes all creatures, but squids are apparently not creatures according to Bukkit. Please create an issue at github.com/TotalFreedom/TotalFreedomMod/issues.

### PR DESCRIPTION
**Pull request by mayokek**

### CODE REVIEW

**Comment by ghost** [comment]
Yes, this works, have tested before.

**Review by ghost — APPROVED** [review]
Simple code change, I have tested this and it works, approved.

**Comment by PacksGamingHD** [comment]
@Commodore64x, "Only those with write access to this repository can merge pull requests.". You can just approve something

**Comment by Commodore64x** [comment]
@PacksGamingHD I didn't try to merge it... Notice how I approved it lmao... I don't understand your comment in relation to my most recent post.

**Comment by mayokek** [comment]
@PacksGamingHD, he approved my changes and tested it. He knows that he can't merge it.

**Comment by PacksGamingHD** [comment]
I’m aware. I’m just saying that it’s not his job to approve the PR's

**Comment by Commodore64x** [comment]
Packs I have only approved the code because I reviewed it and found no errors, I am not approving it for merge, etc.

**Comment by PacksGamingHD** [comment]
But you can’t approve anything

**Comment by mayokek** [comment]
@PacksGamingHD, do a review on the code changes and see.

**Comment by PacksGamingHD** [comment]
I’m aware that they work, but Jerom is the one who has the power to accept PRs. Unless I’ve been under a rock and you’ve suddenly become TFM devs or GitHub moderators

**Review by ghost — CHANGES_REQUESTED** [review]
<Removed by Github Moderator>

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_mobpurge.java
Comment (ghost) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_mobpurge.java]: Replace ent instanceof Squid with ent instanceof WaterMob

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_mobpurge.java
Comment (ghost) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_mobpurge.java]: Add import org.bukkit.entity.WaterMob and remove org.bukkit.entity.Squid

**Review by ghost — CHANGES_REQUESTED** [review]
Changes requested.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_mobpurge.java
Comment (mayokek) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_mobpurge.java]: Does it work?

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_mobpurge.java
Comment (mayokek) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_mobpurge.java]: Read my comment above.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_mobpurge.java
Comment (ghost) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_mobpurge.java]: Yes it does, it finds all watermobs not just squid.

Comment (mayokek) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_mobpurge.java]: Okay, I will make the changes tonight and test it.

**Comment by ghost** [comment]
Add the needs changes tag please @Wild1145

**Comment by ghost** [comment]
Needs review @JeromSar

**Comment by Wild1145** [comment]
@Pramire This has been reviewed and is awaiting changes, it will be reviewed again when the changes are made. You do not need to bump the issue.

**Review by Wild1145 — CHANGES_REQUESTED** [review]
Please make the changes that @Commodore64x has suggested. This should then be in a position to be tested and merged.

**Comment by AvalancheYT** [comment]
Testing watermobs now.

**Comment by mayokek** [comment]
Okay, latest change have been made on request. Tested and the code is working.

**Review by Wild1145 — APPROVED** [review]

**Comment by Wild1145** [comment]
Thanks for contributing. I am closing this issue for inactivity. Should you have anything further to add in the future please either re-open the issue yourself or tag a member of staff and we will re-open the issue for further discussion.

**Pull request closed without merging**
